magprime/templates/emails/precon_faqs.html

```diff
@@ -8,23 +8,24 @@
+Yes, but there is a possibility of us selling out, so check our website for any ticket availability alerts. Full four-day passes will be available for $80, and one day passes will be available for $40 for Thursday, $45 for Friday, $45 for Saturday, and $25 for Sunday.</p>
 <p><b>Where do I go when I arrive? How do I pick up my pass?</b><br />
-The first step is to go to the registration desk, located in Expo Hall B to pick up your pass. The registration desk is open 24 hours a day from Friday-Monday, and will also be open Thursday evening for people who pre-registered. Follow the green line in <a href="http://magfest.org/sites/default/files/M13DirectionsToRegistration.jpg" target="_blank"><b>this map</b></a> to help find your way to registration from any of the three entrances to the hotel. If you pre-registered, simply show photo id such as a driver's license, military id, or student id to the registration staffers. If you do not have photo id, print out the confirmation e-mail you received when you pre-registered.</p>
+The first step is to go to the registration desk, located in Expo Hall E to pick up your pass. The registration desk is open 24 hours a day from Thursday 10am - Sunday 3pm. Follow the green line in <a href="http://magfest.org/static/images/maps/m16toreg.png" target="_blank"><b>this map</b></a> to help find your way to registration from any of the three entrances to the hotel. If you pre-registered, simply show photo id such as a driver's license, military id, or student id to the registration staffers. If you do not have photo id, print out the confirmation e-mail you received when you pre-registered.</p>
 <p><b>Which registration line do I use?</b><br />
-- If you are not already registered for a pass, use the self-service registration and payment stations located outside Expo Hall B. Cash and Credit are accepted. Once you are registered and paid, proceed to badge pickup inside Expo Hall B.<br />
-- Staff, Supporters, Guests, Vendors, Indie showcase exhibitors, and Panelists should use the VIP Badge Pickup line inside Expo Hall B<br />
-- All other attendees (including those who purchased group registrations, or those who registered at-door) should use the Standard Badge Pickup line inside Expo Hall B</p>
+- If you are not already registered for a pass, you can <a href="http://magfest.org/register" target="_blank"><b>purchase one online</b></a>, or use the self-service registration and payment stations located on the left side of Expo Hall E. Cash and Credit are accepted. Once you are registered and paid, proceed to badge pickup inside Expo Hall E.<br />
+- Staff, Supporters, Guests, Vendors, Indie showcase exhibitors, and Panelists should use the VIP Badge Pickup line inside Expo Hall E<br />
+- All other attendees (including those who purchased group registrations, press, or those who registered at-door) should use the Standard Badge Pickup line inside Expo Hall E</p>
```

Comment:
Would you like me to add some if/else statements here?  Since we know precisely what type of badge they have, we can ensure that we always say the correct thing, e.g. if they're a dealer we can say that they should go to the dealer line, etc.  No need to say "if you're a ____" when we can do that in code and do the thinking for them :)